otpInput/otpInput.js

```diff
@@ -0,0 +1,148 @@
+export class OtpInput {
+  form = document.createElement('form');
+  inputQty = 6;
+  inputs = [];
+  input = document.createElement('input');
+  otp = '';
+  inputMatchesOtp = false;
```

Comment:
Can we just define these inside of constructor ? I think that's considered a best practice for JS classes

Reply:
all updates made